sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResourceImpl.java

```diff
@@ -63,7 +65,7 @@ public Response getRowWithWhere(
       final String sortJson,
       final HttpServletRequest request) {
     if (isStringEmpty(where)) {
-      return jaxrsBadRequestError("where parameter is required").build();
+      throw new WebApplicationException("where parameter is required", Status.BAD_REQUEST);
```

Comment:
IMHO this equivalent in terms of readability and verbosity. In addition:
- it makes control flow easier if you want to fail from a deep code path
- I think it will be handled better when we switch to async (to be verified, but I think throwing from a method will complete the async request, so we won't need to change the code).

Reply:
Yea, but usually you want to avoid the exception throwing for control flows, as it's come with the perf penalty for constructing and throwing the exception (like strack trace gathering).. But to me it seems that `where` should be always set, so we are not on the "hot" path here..

Reply:
Agreed, I wouldn't do it for regular logic, but for error handling it feels acceptable.